modules/cudafeatures2d/include/opencv2/cudafeatures2d.hpp

```diff
@@ -434,11 +434,17 @@ class CV_EXPORTS FastFeatureDetector : public cv::FastFeatureDetector, public Fe
 
         FEATURE_SIZE = 7
     };
+    enum
+    {
+        TYPE_5_8 = 0, TYPE_7_12 = 1, TYPE_9_16 = 2,
+        THRESHOLD = 10000, NONMAX_SUPPRESSION = 10001, FAST_N = 10002,
```

Comment:
Code duplication should be avoided.

Does "using FastFeatureDetector::TYPE_5_8;" work here?

Reply:
I tried it, but it says `error C2602: 'TYPE_9_16' is not a member of a base class of 'cv::cuda::FastFeatureDetector'`.
We can either duplicating the enum or add `cv::` to the default value, thus become `cv::FastFeatureDetector::TYPE_9_16`. What do you think?